### PR TITLE
Add damp scores to status command

### PR DESCRIPTION
**Pull request by jwolski**
This PR adds a DAMPSCORE column to the status command which takes the min and max score of each member:

```
 172.18.24.128:3000   alive   0..4568
 172.18.24.128:3001   alive   0..0
 172.18.24.128:3002   alive   0..0
 172.18.24.128:3003   alive   0..0
 172.18.24.128:3004   alive   0..0
```

@uber/ringpop 

Reviewers: @dansimau

### CODE REVIEW

File: parser.js [hunk 1]
```diff
@@ -49,12 +49,16 @@ function parseReuseCommand() {
 function parseStatusCommand() {
     program
         .description('Status of members in ring')
+        .option('-q, --quiet', 'Do not print headers')
```
Comment (dansimau) [inline parser.js]: Is the `-q` option needed?

File: lib/util.js [hunk 2]
```diff
@@ -19,6 +19,12 @@
 // THE SOFTWARE.
 'use strict';
 
+function assertTruthy(truthy, message) {
```
Comment (dansimau) [inline lib/util.js]: loljs

File: status.js [hunk 3]
```diff
@@ -21,33 +21,61 @@
...
-            process.exit(1);
-        }
-
-        var table = createTable([]);
-        var cluster = clusterManager.getClusterAt(0);
-        cluster.membership.forEach(function each(member) {
-            table.push([member.address, member.status]);
+    cluster.fetchStats(function onStats(err) {
+        assertTruthy(!err, (err && err.message));
+        assertTruthy(cluster.getPartitionCount() === 1,
```
Comment (dansimau) [inline status.js]: A discussion outside the scope of this PR, but we should talk about whether the `status` command can tell us something useful even when a ring is partitioned. I previously found it weird that the `status` command only gives the status if the ring is healthy. Part of the reason you might want the status is if you know your ring is unhealthy and you're trying to get more information.

Comment (jwolski) [inline status.js]: This is true.

This is a larger question for ringpop-admin for sure. Honestly, the only reason it was done this way was due to time constraints.

`top` gives you more information about a unhealthy cluster than the others. `count` and `checksums` will give you the some idea about its unhealthiness, but we should talk about the best way to represent a non-converged cluster.

**Comment by dansimau** [comment]
I think we should have tests in every PR for ringpop-admin moving forward. But other than that, looks great. Stellar effort. A+++ would buy again :+1:

File: parser.js (hunk 1 above)
Comment (jwolski) [inline parser.js]: *shrug* if people want to not see the column headers??

**Comment by jwolski** [comment]
yes... tests.......................................................................................................................

File: lib/util.js (hunk 2 above)
Comment (rf) [inline lib/util.js]: why not use `require('assert')` here? seems functionally equivalent

Comment (jwolski) [inline lib/util.js]: aside from the big ugly stack trace, this is true.

**Pull request merged**